privateca/snippets/src/test/java/privateca/SnippetsIT.java

```diff
@@ -112,14 +109,15 @@ public static void setUp()
     KEY_SIZE = 2048; // Default key size
 
     // Delete stale resources
-    Util.cleanUpCaPool(PROJECT_ID, LOCATION);
-    TimeUnit.SECONDS.sleep(30);
+//    Util.cleanUpCaPool(PROJECT_ID, LOCATION);
+//    TimeUnit.SECONDS.sleep(30);
```

Comment:
Kindly uncomment this. I believe this deletes any stale CA pools leftover from previous tests.